ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml

```diff
@@ -94,17 +94,18 @@ tests:
     - chain: openshift-e2e-test-qe
     workflow: openshift-e2e-aws-single-node-recert
   timeout: 8h0m0s
-- as: aws-sno-fips-recert
+- as: e2e-aws-ipi-sno-fips-recert
```

Comment:
cc @liangxia to review.

Reply:
What's reason to rename this test name with prefix `e2e` ?

Reply:
The added execution test is `openshift-e2e-test-qe`,  so named with prefix e2e.

Reply:
Just found this is another team's repo/folder. I don't have objection to this. No change required.